index.html

```diff
@@ -688,6 +751,60 @@
         <li>When the request has been completed, resolve <var>promise</var> with <code>true</code>.
         </li>
       </ol>
+      <p>
+        The serializer for a <a><code>PushSubscription</code></a> invokes the following steps:
+      </p>
+      <ol>
+        <li>Let <var>map</var> be an empty map.
+        </li>
+        <li>Add an entry to <var>map</var> whose key name is <code>endpoint</code> and whose value
+        is the result of converting the <code>endpoint</code> attribute of the
+        <a><code>PushSubscription</code></a> to a serialized value.
+        </li>
+        <li>Let <var>keys</var> be an empty map.
+        </li>
+        <li>For each identifier <var>i</var> corresponding to keys in internal slots on the
+        <a><code>PushSubscription</code></a>, ordered by the name of the key:
+          <ol>
+            <li>Let <var>b</var> by the encoded value of the public key corresponding to the key
+            name <var>i</var>, using the encoding defined for the key name (see <code><a data-lt=
+            "PushSubscription.getKey">getKey</a></code>).
+            </li>
+            <li>Let <var>s</var> be the URL-safe base64 encoding of <var>b</var> as a
```

Comment:
Maybe.  There is no inherent reason that base64 can't be constant time, though I'll note that it commonly isn't.